client/src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule, Routes } from '@angular/router';
+
+import { SharedModule } from '../shared/shared.module';
+
+import { LoginFormComponent } from './login-form/login-form.component';
+
+const routes: Routes = [
+  { path: 'login', component: LoginFormComponent }
+]
+
+@NgModule({
+  declarations: [
+    LoginFormComponent
+  ],
+  imports: [
+    CommonModule,
+    SharedModule,
+    ReactiveFormsModule,
+    RouterModule.forChild(routes)
```

Comment:
usually for child is used for lazy loaded modules

Reply:
I moved this logic to app routing module